app/team.go

```diff
@@ -1002,8 +1006,7 @@ func (a *App) LeaveTeam(team *model.Team, user *model.User, requestorId string)
 		}
 	}
 
-	err = a.RemoveTeamMemberFromTeam(teamMember, requestorId)
-	if err != nil {
+	if err = a.RemoveTeamMemberFromTeam(teamMember, requestorId); err != nil {
```

Comment:
`:=`

Reply:
@agnivade  why? there's no shadowing here

Reply:
Just to make it more idiomatic and consistent with the other if blocks. We are declaring a new scope with the if block. But reusing the `err` from the parent scope. All other if blocks like this do the same.